crates/dt-workspace/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use std::{
+    borrow::Cow,
+    path::{Path, PathBuf},
+};
+
+use config::{Config, ConfigError};
+use thiserror::Error;
+
+pub mod config;
+mod search;
+
+#[derive(Debug)]
+pub struct Workspace {
+    path: PathBuf,
+
+    config: Config,
+}
+
+impl Workspace {
+    pub fn try_new(override_path: Cow<'static, Path>) -> Result<Self, WorkspaceError> {
```

Comment:
```suggestion
    pub fn try_new(path: Cow<'static, Path>) -> Result<Self, WorkspaceError> {
```
Because it's no longer optional, the prefix doesn't fit.